src/main/java/io/employees/HomeWork/EmployeeController.java

```diff
@@ -0,0 +1,32 @@
+package io.employees.HomeWork;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/employee")
+public class EmployeeController {
+    private final EmployeeServiceImpl employeeServiceImpl;
+
+    public EmployeeController(EmployeeServiceImpl employeeServiceImpl) {
```

Comment:
```suggestion
    public EmployeeController(EmployeeService employeeServiceImpl) {
```
инъекцию надо делать по интерфейсу, а не реализации